tools/Dockerfile.kapitan

```diff
@@ -1,9 +1,13 @@
-# Pushed to docker.io/vshn/kapitan:v0.26.0
+# Pushed to docker.io/vshn/kapitan:v0.27.2
 
 FROM docker.io/alpine:3.11
 
-ENV KAPITAN_VERSION=v0.26.0
+ENV KAPITAN_VERSION=v0.27.2
 
-RUN apk add --no-cache curl \
+RUN apk add --no-cache \
+        ca-certificates \
+        curl \
  && curl -Lo /usr/local/bin/kapitan https://github.com/deepmind/kapitan/releases/download/${KAPITAN_VERSION}/kapitan-linux-amd64 \
  && chmod +x /usr/local/bin/kapitan
+
+USER 10001
```

Comment:
Is 10k+1 the correct user id here? Or did you mean 1001?

Reply:
Doesn't really matter, as long as it's higher than 1000